lib/assessment.js

```diff
@@ -4,24 +4,43 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
 
-};
+//arguments = "array-ish"
+var min = function min() {
+  var result = arguments[0]
+  for(var index; index<arguments.length; index ++)
+    if(arguments[index]<result) {
+      result=arguments[index]
+    }
+return result
+}`
+//I used mdn
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-var sum = function sum() {
 
-};
+var sum = function sum(){
+  var result = 0;
+    for (var index = 0; index < args.length; index ++) {
+      result = result + args[index];
```

Comment:
`arguments`, not `args`. I was abbreviating it to save myself some wrist motions.